examples/src/pages/Viewer.tsx

```diff
@@ -175,7 +174,7 @@ export function Viewer() {
 
 
       const totalBounds = new THREE.Box3();
-      function handleModelAdded(model: CogniteModelBase) {
+      function handleModelAdded(model: Cognite3DModel | CognitePointCloudModel) {
```

Comment:
I think it would make sense to create a type called something like `CogniteModel` that will have the same value of `Cognite3DModel | CognitePointCloudModel`. It looks a little more cleaner for me. Applies to all places where `CogniteModelBase` was previously.

Reply:
@Savokr Should we rename `Cognite3DModel `to `CogniteCadModel `and then have the joint type just be `CogniteModel`?

Reply:
Yeah, agree

Reply:
Rename completed.